web-next/app/globals.css

```
@@ -0,0 +1,960 @@
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  gap: 10px;
+  font-size: 11px;
+  color: var(--txt3);
+  max-width: 1440px;
+  margin: 0 auto;
+  font-family: var(--font-mono), "JetBrains Mono", monospace;
+}
+
+@keyframes fadeUp {
```

Stylelint (17.6.0)

[error] Expected keyframe name "fadeUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)